src/services/ContributingDeveloperAuditService/ContributingDeveloperAuditService.ts

```diff
@@ -0,0 +1,87 @@
+import SOOSHooksApiClient, { IContributorAuditModel } from "../../api/SOOSHooksApiClient";
+import { SOOS_CONSTANTS } from "../../constants";
+import { ScmType } from "../../enums";
+import { soosLogger } from "../../logging";
+import GitHubAudit from "./scm/GitHub/GitHubAudit";
+import FileSystem from "fs";
+import * as Path from "path";
+import { ParamUtilities } from "./utilities";
+
+export interface IContributingDeveloperAuditProvider {
```

Comment:
we should make this match the api endpoint since we have the other contributor information gathering for the runnable scripts
```suggestion
export interface IContributorAuditProvider {
```